Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Runs master branch from github of lando/lando-messaging
 # Requires a lando config file mounted at /etc/lando_config.yml.
 FROM python:2.7.13
-RUN pip install --upgrade git+git://github.com/Duke-GCB/lando-messaging.git
 ADD . /src
 WORKDIR /src
+RUN pip install --upgrade git+git://github.com/Duke-GCB/lando-messaging.git
```

Comment:
Only other thing I could think to do is include lando-messaging as a git submodule, which would require a commit to lando's repo to update lando-messaging